benchmarks/pipe.py

```diff
@@ -284,7 +294,7 @@ def train(data_config, model, benchmark_config, args):
         if pipe_group is None or pipe_group.rank() == pipe_group.size() - 1:
             total_loss += loss.item()
             log_interval = 1
-            word_counter += batch["ntokens"]
+            word_counter += np.prod(list(batch["input"].size()))
```

Comment:
again, I think batch["input"].numel() should do.

total_words and word_counter are really similar. if they are different counters, I think we should rename them to make sure their names convey the difference.